packages/office-addin-telemetry/src/telemetryJsonData.ts

```diff
@@ -0,0 +1,107 @@
+            return !groupNameExists(jsonData, groupName);
+        }
+        return true;
+    } catch (err) {
+        console.log(chalk.default.red(err));
+    }
+}
+/**
+ * Allows developer to add or modify a specific property to the group
+ * @param groupName Group name of property
+ * @param property Property that will be created or modified
+ * @param value Property's value that will be assigned
+ * @param jsonFilePath Optional path to the json config file
+ */
+export function modifyTelemetryJsonData(groupName: string, property: any, value: any, jsonFilePath = telemetryJsonFilePath): void {
+    try {
+        if (fs.existsSync(jsonFilePath)) {
+            const telemetryJsonData = readTelemetryJsonData(jsonFilePath);
+            if (groupNameExists(telemetryJsonData, groupName)) {
+            telemetryJsonData.telemetryInstances[groupName][property] = value;
+            fs.writeFileSync(jsonFilePath, JSON.stringify((telemetryJsonData), null, 2));
+            }
+        }
+    } catch {
+        console.log(chalk.default.red("Could not modify property!\n Make sure the group name and file exists!"));
+    }
+}
+/**
+ * Reads data from the telemetry json config file
+ * @param jsonFilePath Optional path to the json config file
+ * @returns Parsed object from json file if it exists
+ */
+export function readTelemetryJsonData(jsonFilePath = telemetryJsonFilePath): any {
+    if (fs.existsSync(jsonFilePath)) {
+        const jsonData = fs.readFileSync(jsonFilePath, "utf8");
+        return JSON.parse(jsonData.toString());
+    }
+}
+/**
+ * Returns telemetry level of telemetry object
+ * @param groupName Group name to search for in the specified json data
+ * @param jsonFilePath Optional path to the json config file
+ */
+export function readTelemetryLevel(groupName: string, jsonFilePath = telemetryJsonFilePath): string {
+        const jsonData = readTelemetryJsonData(jsonFilePath);
+        return jsonData.telemetryInstances[groupName].telemetryObjectLevel;
+}
+/**
+ * Returns telemetry property of telemetry object
+ * @param groupName Group name to search for in the specified json data
+ * @param propertyName Property name that will be used to access and return the associated value
+ * @param jsonFilePath Optional path to the json config file
+ */
+export function readTelemetryObjectProperty(groupName: string, propertyName: string, jsonFilePath = telemetryJsonFilePath): string {
+    const jsonData = readTelemetryJsonData(jsonFilePath);
+    return jsonData.telemetryInstances[groupName][propertyName];
+}
+/**
+ * Writes to telemetry config file either appending to already existing file or creating new file
+ * @param groupName Group name of telemetry object
+ * @param telemetryLevel Whether user is sending basic or verbose telemetry
+ * @param jsonFilePath Optional path to the json config file
+ */
+
+export function writeTelemetryJsonData(groupName: string, telemetryObjectLevel: string, jsonFilePath = telemetryJsonFilePath): void {
```

Comment:
let's just call this telemetryLevel instead of telemetryObjectLevel.  Also, the type should be telemetryLevel (the enum) and not string